src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -434,4 +450,18 @@ public String toString() {
     /* Each datacenter requires at minimum one reaper instance that has jmx access to all nodes in that datacenter */
     EACH
   }
+
+  public static class AccessControlConfiguration {
```

Comment:
nit-pick: can the inner class be `final`?

Reply:
Sure thing